src/runtime-rs/crates/hypervisor/Cargo.toml

```diff
@@ -14,6 +14,8 @@ dbs-utils = "0.1.0"
 go-flag = "0.1.0"
 libc = ">=0.2.39"
 nix = "0.24.2"
+openssl = { version = "0.10.35", features = ["vendored"] }
+# With `openssl-src`, openssl wil be compiled statically and we don't have to install openssl-dev manually anymore.
```

Comment:
Please use openanolis/dragonball-sandbox#219 to support this feature.

Reply:
fixed